scripts.js

```diff
@@ -1,5 +1,9 @@
 function tipCalculate (slider, bill){
-    
```

Comment:
I moved the logic from this function into an identical function named "updateTip" and call into that function here. Then I add the additional line which sets the value of the slider. Moving this logic was done so that the slider can call into updateTip without triggering the additional line that sets its value. If the slider were to call tipCalculate then the extra line that changes the value could trigger an additional call back into tipCalculate. (This would create recursion which we don't want.)